connect/runtime/src/test/java/org/apache/kafka/connect/runtime/standalone/StandaloneHerderTest.java

```diff
@@ -722,89 +646,68 @@ public void testRestartConnectorAndTasksBoth() throws Exception {
         FutureCallback<ConnectorStateInfo> restartCallback = new FutureCallback<>();
         herder.restartConnectorAndTasks(restartRequest, restartCallback);
         assertEquals(connectorStateInfo, restartCallback.get(1000L, TimeUnit.MILLISECONDS));
-        PowerMock.verifyAll();
+
+        verifyConnectorStatusRestart();
+        verify(statusBackingStore).put(taskStatus.capture());
```

Comment:
Can we also verify the state and task ID? E.g.:
```java
assertEquals(AbstractStatus.State.RESTARTING, taskStatus.getValue().state());
assertEquals(taskId, taskStatus.getValue().id());
```

Reply:
Commited and pushed